### PR TITLE
Travis job parallelization 

### PR DESCRIPTION
**Pull request by patjouk**
Closes MozillaFoundation/mofo-devops#655

WIP to parallelize our tests on Travis.

## Checklist

**Documentation:**
- [x] Is my code documented?
- [x] Did I update the READMEs or wagtail documentation?

### CODE REVIEW

**Comment by patjouk** [comment]
I tried to make the linter errors more visible with colors, but it didn't work :( I added the `--statistics`, as a tiny improvement: https://travis-ci.org/mozilla/foundation.mozilla.org/jobs/499332478#L491-L493

**Comment by Pomax** [comment]
This is cool! Are we limited in the number of Travis runs, or can we make a dedicated "linting" task that that run separately from `manage test` and visual regression (each in their own task?)

**Comment by patjouk** [comment]
We can go to 200 jobs max: Percy will definitely run on a separate job. At some point, I was thinking of doing a fail fast for linters and play around stages. I'll experiment on that once I've brought back Percy.

**Comment by patjouk** [comment]
Finally ready for review: I tried to do something around github labels but it didn't work and I'm not sure it was a good idea in the end.

Changes:
- Dependabot PRs won't trigger a percy run (close #2719). example: https://travis-ci.org/mozilla/foundation.mozilla.org/builds/503122701
- Job matrix: python tests, npm tests and percy run on different jobs: travis speed is slightly increased.

File: .editorconfig [hunk 1]
```diff
@@ -11,3 +11,6 @@ trim_trailing_whitespace = true
 
 [*.py]
 indent_size = 4
+
+[Makefile]
```
Comment (cadecairos) [inline .editorconfig]: Let's drop this

**Pull request merged**